posts/2022/10/19/getting-started-with-event-driven-ansible.md

```diff
@@ -0,0 +1,174 @@
+Event-Driven Ansible is currently in developer preview, however there is nothing stopping us from installing ansible-rulebook, which is the CLI component of Event-Driven Ansible, and building our first rulebook. Event-Driven Ansible contains a decision framework that was built using Drools. We need a rulebook to tell the system what events to flag and how to respond to them. These rulebooks are also created in YAML and are used like traditional Ansible Playbooks, so this makes it easier to understand and build the rulebooks we need. One key difference between playbooks and rulebooks is the If-this-then-that coding that is needed in a rulebook to make an event driven  automation approach work.  
+
+A rulebook is comprised of three main components: 
+
+* **Sources** define which event source we will use. These sources come from source plugins which have been built to accommodate common use cases. With time, more and more sources will be available. There are some source plugins that are available already, including: webhooks, Kafka, Azure service bus, file changes, and alertmanager. 
+
+* **Rules** define conditionals we will try to match from the event source. Should the condition be met, then we can trigger an action.
+
+* **Actions** trigger what you need to happen should a condition be met. Some of the current actions are: run_playbook, run_module, set_fact, post_event, and debug.
+
+![getting-started-with-event-driven-ansible](/images/posts/getting-started-with-event-driven-ansible/event-source.png)
+
+Now, let's install ansible-rulebook and start with our very first event.
+
+To install ansible-rulebook, we can install our Galaxy Collection, which has a playbook to install everything we need.
+
+`ansible-galaxy collection install ansible.eda`
+
+Once the Collection is installed, you can run the install-rulebook-cli.yml playbook. This will install everything you need to get started with ansible-rulebook on the command line. This is currently supported for Mac and Fedora.
+
+**Note:** Now, you could also skip this method above and install ansible-rulebook with pip, followed by installing the ansible.eda collection. Java 11+ is required if you use this method and we suggest using [openjdk](https://jdk.java.net/19/). (This step is not required if you used the previous install method.)
+~~~
+`pip install ansible-rulebook`
+
+`ansible-galaxy collection install ansible.eda`
+~~~
+If you want to contribute to ansible-rulebook, you can also fork the following [GitHub repository](https://github.com/ansible/ansible-rulebook). This repository also contains instructions for setting up your development environment and how to build a test container. 
+
+Let's build an example rulebook that will trigger an action from a webhook. We will be looking for a specific payload from the webhook, and if that condition is met from the webhook event, then ansible-rulebook will trigger the desired action. Below is our example rulebook:
+
+```
+…webhook-example.yml
```

Comment:
Although this line is in the original, I would move it before the code box to avoid confusing people.